react-validated-input-component/src/validate-input.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+
+export default function ValidateInput() {
+
+  const switchClasses = {
+    hidden: 'hidden',
+    greenCheck: {
```

Comment:
For maintainability, it might be best to name these after their purpose instead of after what they look like. For example, `valid` and `invalid`. That way, if you decide to change the styling from a green check and a red X, you won't have to rename these.